### PR TITLE
Make minor documentation improvements to Generators section

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by hrishin** [comment]
Thanks a lot, @geoand. It looks better :+1:

**Comment by geoand** [comment]
You are welcome @hrishin!

File: doc/src/main/asciidoc/inc/generator/_overview.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-The usual way to define Docker images is with the plugin configuration as explained in <<fabric8:build>>. This can either be done completely within the `pom.xml` or by refering to an external Dockerfile. Since fabric8-maven-plugin includes https://github.com/fabric8io/docker-maven-plugin[docker-maven-plugin] the way how images are built is identical.
+The usual way to define Docker images is with the plugin configuration as explained in <<fabric8:build>>. This can either be done completely within the `pom.xml` or by refering to an external Dockerfile. Since fabric8-maven-plugin includes https://github.com/fabric8io/docker-maven-plugin[docker-maven-plugin] the way which images are built is identical.
```
Comment (rohanKanojia) [inline doc/src/main/asciidoc/inc/generator/_overview.adoc]: s/refering/referring/

File: doc/src/main/asciidoc/inc/generator/_java_exec.adoc [hunk 2]
```diff
@@ -56,11 +56,11 @@ Beside the common configuration parameters described in the table <<generator-op
 | 8778
 
 | *mainClass*
-| Main class to call. If not given first a check is performed to detect a fat-jar (see above). Next a class is tried to be found by scanning `target/classes` for a single class with a main method. If no if found or more than one is found, then this generator does nothing.
+| Main class to call. If not given first a check is performed to detect a fat-jar (see above). Next a class is looked up by scanning `target/classes` for a single class with a main method. If no such class is found or if more than one are found, then this generator does nothing.
```
Comment (rohanKanojia) [inline doc/src/main/asciidoc/inc/generator/_java_exec.adoc]: s/one are found/one is found/

File: doc/src/main/asciidoc/inc/generator/_java_exec.adoc [hunk 3]
```diff
@@ -2,16 +2,16 @@
 === Java Applications
 
 One of the most generic _Generators_ is the `java-exec` generator.
-It is responsible to start up arbitrary Java application.
+It is responsible for starting up arbitrary Java application.
 It knows how to deal with fat-jar applications where the application and all dependencies are included within a single jar and the `MANIFEST.MF` within the jar references a main class.
 But also flat classpath applications, where the dependencies are separate jar files and a main class is given.
 
-If no main class is explicitely configured, first a fat jar is tried to be found.
+If no main class is explicitily configured, the plugin first attempts to locate a fat jar.
```
Comment (rohanKanojia) [inline doc/src/main/asciidoc/inc/generator/_java_exec.adoc]: s/explicitily/explicitly

**Comment by geoand** [comment]
@rohanKanojia I updated the PR with the changes fixes you spotted. Thanks!

**Comment by geoand** [comment]
@rohanKanojia Thank you!

**Comment by piyush1594** [comment]
@geoand Please take a look, changes are not reflected in the PR.

File: doc/src/main/asciidoc/inc/generator/_overview.adoc (hunk 1 above)
Comment (piyush1594) [reply doc/src/main/asciidoc/inc/generator/_overview.adoc]: s/ the way which/, the way by which/

**Comment by geoand** [comment]
@piyush1594 The changes should be visible now

**Comment by piyush1594** [comment]
Thanks @geoand, this looks good.

**Comment by geoand** [comment]
@piyush1594 Glad to hear it!

**Pull request merged**